event.go

```diff
@@ -467,11 +469,44 @@ func (e Event) PrevEvents() []EventReference {
 	return e.fields.PrevEvents
 }
 
+// PrevEventIDs returns the event IDs of the direct ancestors of the event.
+func (e Event) PrevEventIDs() []string {
+	result := make([]string, len(e.fields.PrevEvents))
+	for i := range e.fields.PrevEvents {
+		result[i] = e.fields.PrevEvents[i].EventID
+	}
+	return result
+}
+
+// Membership returns the value of the content.membership field if this event
+// is an "m.room.member" event.
+// Returns an error if the event is not a m.room.member event or if the content
+// is not valid m.room.member content.
+func (e Event) Membership() (*string, error) {
```

Comment:
I'd prefer this to be `(string, error)` on the basis that `""` is not a valid membership state, and constantly dereffing is annoying.